specs/_features/maxeb_increase/capella.py

```diff
@@ -1402,7 +1445,7 @@ def get_eligible_validator_indices(state: BeaconState) -> Sequence[ValidatorInde
     previous_epoch = get_previous_epoch(state)
     return [
         ValidatorIndex(index) for index, v in enumerate(state.validators)
-        if is_active_validator(v, previous_epoch) or (v.slashed and previous_epoch + 1 < v.withdrawable_epoch)
+        if is_active_validator(v, previous_epoch) or (is_slashed_attester(v) and previous_epoch + 1 < v.withdrawable_epoch)
```

Comment:
Same here, would replace with this:

```suggestion
        if is_active_validator(v, previous_epoch) or (is_slashed_validator(v) and previous_epoch + 1 < v.withdrawable_epoch)
```